pyvista/plotting/plotter.py

```diff
@@ -6652,25 +6772,25 @@ class Plotter(BasePlotter):
 
     def __init__(
         self,
-        off_screen=None,
-        notebook=None,
-        shape=(1, 1),
-        groups=None,
-        row_weights=None,
-        col_weights=None,
-        border=None,
-        border_color='k',
-        border_width=2.0,
-        window_size=None,
-        line_smoothing=False,
-        point_smoothing=False,
-        polygon_smoothing=False,
-        splitting_position=None,
-        title=None,
-        lighting='light kit',
-        theme=None,
-        image_scale=None,
-    ):
+        off_screen: bool | None = None,
+        notebook: bool | None = None,
+        shape: Sequence[int] | str = (1, 1),
+        groups: Sequence[int] | None = None,
+        row_weights: Sequence[int] | None = None,
+        col_weights: Sequence[int] | None = None,
+        border: bool | None = None,
+        border_color: ColorLike = 'k',
+        border_width: float = 2.0,
+        window_size: list[int] | None = None,
+        line_smoothing: bool = False,
+        point_smoothing: bool = False,
+        polygon_smoothing: bool = False,
+        splitting_position: float | None = None,
+        title: str | None = None,
+        lighting: Literal['light kit', 'three lights', 'none'] = 'light kit',
```

Comment:
This is repeated elsewhere
```suggestion
        lighting: LightingOptions = 'light kit',
```